Makefile

```diff
@@ -0,0 +1,31 @@
+# define the name of the virtual environment directory
+VENV := venv
+
+# default target, when make executed without arguments
+all: venv
+
+$(VENV)/bin/activate: requirements.txt
+	python3 -m venv $(VENV)
+	./$(VENV)/bin/pip install -r requirements.txt
+	./$(VENV)/bin/pip install pandas
```

Comment:
`pandas` is not required, let's move it to a separate `make` command. Also, `pandas` installation is a part of plugin's `extras`: https://github.com/bryzgaloff/airflow-clickhouse-plugin/blob/master/setup.py#L31, so better rely on it instead of running the `pandas` installation manually.

Reply:
Yes, pandas is an extra package. But for local development, I think, it's needed (e.g. run all tests?)

Reply:
Pandas installation is quite a heavy one, so personally I would prefer to not install it by default. But I am leaving this up to you: feel free to either leave as-is or move to another command (recommended by me).

Reply:
I think, we should give simple way to configure environment and start development without extra commands.